### ISSUE TITLE
Order AOW Email Template dropdown

### ISSUE THREAD

**Issue by willrennie** [salesagility/SuiteCRM#176]
It would be nice if this drop down was sorted alphabetically like all of the other drop downs.

https://suitecrm.com/forum/bug-tracker/4065-send-email-drop-down-in-workflow

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Order AOW Email Template dropdown](https://trello.com/c/UxPtdH4I/157-order-aow-email-template-dropdown)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
